### ISSUE TITLE
Replace Current Pop-up Banner for "Tutorial" Category Blogs

### ISSUE THREAD

**Issue by SourabhEstuary** [estuary/marketing-site#520]
**Objective:** 

To replace the current exit-intent pop-up banner with a new banner that is specifically targeted for visitors of "Tutorial" blog pages. The new banner will showcase Estuary's USP, encouraging users to try Estuary for moving data from "X" to "Y."

**Purpose:**

The main purpose of this pop-up is to make a final attempt to attract users to register on the website and try Estuary for moving data from "X" to "Y." The pop-up will activate when a user’s cursor indicates intent to close the browser tab or window. It will present a clear and compelling message about Estuary’s capabilities, driving them to take action and register for the platform.

**Message to Showcase on Pop-Up**

**Heading:** Instantly Move Your Data – No Code Needed
**Subheading:** Move Data from Any Source to Any Destination in Minutes – No Coding Required

**Key points:**

- Real-time and batch integration in one unified platform
- Enterprise-ready support for all private networking environments
- Load data into multiple destinations simultaneously
- Proven to scale beyond 7GB+/sec, supporting high-volume data

**Button 1:** Try Free for 30 Days  (Primary Button - Should be in highlighted and bold colour)
**Button 2:** Contact Us

**Text below button:** No credit card required

Pop-up example:
![image](https://github.com/user-attachments/assets/47493a3f-8183-4afa-bc05-c34cc6cfa522)

**Comment by Brenosalv** [comment]
@altitude-lucas Can we reuse UI elements/components we already have on other pages or will we need a customized design for this? Asking because we have the design label in this issue.

**Comment by altitude-lucas** [comment]
@Brenosalv I think it's okay to re-use what we have existing, if there are any specific design elements we're missing we can ask Polina for them but in this case we have most of what we need. @danthelion FYI

**Comment by danthelion** [comment]
This is good to go from my side, let's put it in the backlog @SourabhEstuary

**Comment by Brenosalv** [comment]
@danthelion We need an image to put on the right of the banner. Could you choose one with Polina?

**Comment by SourabhEstuary** [comment]
Hey @danthelion .. need your help on image finalization for the pop-up...

**Comment by danthelion** [comment]
moving to todo

**Comment by Brenosalv** [comment]
@danthelion @SourabhEstuary Will we keep the old popup for the other blog posts that are not Tutorial?

**Comment by Brenosalv** [comment]
Here is a first version of the popup so you can provide a feedback - don't worry about the image, I will adjust it yet:

<img width="1310" alt="Image" src="https://github.com/user-attachments/assets/9ac6917d-da64-4487-8df8-f215d7260a2e" />

**Comment by SourabhEstuary** [comment]
Looks good to me! We can add this. 👍

**Comment by danthelion** [comment]
@SourabhEstuary , can we add this to all blog pages as popup and no just tutorial? I think it's better than the webinar one

**Comment by SourabhEstuary** [comment]
@danthelion Agreed! @Brenosalv Let's replace it in all blogs instead of just the tutorial.
